pkgs/by-name/ku/kubectl-rook-ceph/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-rook-ceph";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "rook";
+    repo = "kubectl-rook-ceph";
+    rev = "v${version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
We use `tag` now when dealing with tags.